src/const.ts

```diff
@@ -108,3 +108,5 @@ export const INPUT_PRECISION_SIZE = 6
 export const VALID_UNTIL_DEFAULT = '2880'
 export const VALID_FROM_DEFAULT = '30'
 export const WETH_ADDRESS_MAINNET = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
+export const ORDER_BOOK_HOPS_DEFAULT = 2
+export const ORDER_BOOK_HOPS_MAX = 2
```

Comment:
oh, so only 1 or 2? what a let down...

Reply:
Ahaha, I double checked with @fleupold , if this changes he will notify us, but for now 2 hops tops. I'm not sure if you would get a great difference with more hops in the book, but sure there would be some CPU bill to pay by devops.

I think it's interesting to set it to 0, and see how it changes already with 1 hop.

This is a hidden functionality for debugging, and impress people in the bar 🍻